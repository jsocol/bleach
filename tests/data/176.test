<div id="108"><!-- IE 5-8 standards mode -->
<a href=http://foo.bar/#x=`y></a><img alt="`><img src=xx onerror=alert(108)></a>">
<!-- IE 5-9 standards mode -->
<!a foo=x=`y><img alt="`><img src=xx onerror=alert(2)//">
<?a foo=x=`y><img alt="`><img src=xx onerror=alert(3)//">//["'`-->]]>]</div>
