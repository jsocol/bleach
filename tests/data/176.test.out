<div id="108">
<a href="http://foo.bar/#x=`y"></a><img alt="`&gt;&lt;img src=xx onerror=alert(108)&gt;&lt;/a&gt;">

<img alt="`&gt;&lt;img src=xx onerror=alert(2)//">
<img alt="`&gt;&lt;img src=xx onerror=alert(3)//">//["'`--&gt;]]&gt;]</div>
